action/protocol/execution/testdata/f.value.json

```diff
@@ -29,7 +29,9 @@
         "rawAmount": "0",
         "rawGasLimit": 1200000,
         "rawGasPrice": "0",
-        "failed": true,
+        "failed": false,
```

Comment:
could you explain more? Previously, "failed" was true, did you observe that same result on Ethereum? Now the "failed" value is false, is it the same as Ethereum? If both are yes, which is the real expected value, what's the purpose of this test?